Dockerfile

```diff
@@ -11,10 +11,18 @@ RUN apk add --no-cache samba-dc supervisor \
     && ln -s /samba/lib /var/lib/samba \
     && ln -s /samba/log /var/log/samba
 
+
+# Add 'edge' repos
+RUN echo 'http://dl-cdn.alpinelinux.org/alpine/edge/main' >> /etc/apk/repositories
+RUN echo 'http://dl-cdn.alpinelinux.org/alpine/edge/community' >> /etc/apk/repositories
```

Comment:
What package is `community` required for? Looks like `chrony` is in [`edge/main`](https://pkgs.alpinelinux.org/package/edge/main/x86_64/chrony).